### PR TITLE
feat: improve on table listings (HL-1325)

### PR DESCRIPTION
**Pull request by sirtawast**
## Description :sparkles:

Various small fixes and improvements to help handler on their daily duties. Also fixes some of the issues with listing applicants in new vs. old ahjo mode.

### CODE REVIEW

**Comment by terovirtanen** [comment]
**APPLICANT branch is deployed to platta: https://helsinkilisa-ui-pr3119.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://helsinkilisa-pr3119.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://helsinkilisa-ui-handler-pr3119.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-pr3119.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-handler-pr3119.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://helsinkilisa-pr3119.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-pr3119.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-handler-pr3119.dev.hel.ninja :laughing::tada::tada::tada:

File: frontend/benefit/handler/src/components/applicationList/useApplicationListData.ts [hunk 1]
```diff
@@ -58,25 +59,47 @@ const useApplicationListData = (
...
+        ].includes(application.status) &&
+          isNewAhjoMode &&
+          application.handledByAhjoAutomation) ||
+        ([
+          APPLICATION_STATUSES.ACCEPTED,
+          APPLICATION_STATUSES.REJECTED,
+        ].includes(application.status) &&
+          isNewAhjoMode &&
+          application.handledByAhjoAutomation) ||
+        ([
```
Comment (EmiliaMakelaVincit) [inline frontend/benefit/handler/src/components/applicationList/useApplicationListData.ts:92]: I think these two can be combined together at least to make this huge effective one-liner a bit less wieldy. Could consider just abandoning the idea of combining this all into one statement and using a function with actual if-else structure, too.

**Comment by terovirtanen** [comment]
**HANDLER branch is deployed to platta: https://helsinkilisa-ui-handler-pr3119.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**API branch is deployed to platta: https://helsinkilisa-pr3119.api.dev.hel.ninja/healthz** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-handler-pr3119.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://helsinkilisa-ui-pr3119.dev.hel.ninja :laughing::tada::tada::tada:

**Pull request merged**